main.gms

```diff
@@ -101,9 +111,44 @@
 *' The units (e.g., TWa, EJ, GtC, GtCO2, ...) of variables and parameters are documented in the declaration files.
 *'
 *' For the labels of parameters, scalars and set, use double quotes only.
-
-
-
+*'
+*' #### Commenting:
+*'
+*' * Comment all parts of the code generously
+*' * For all equations, it should become clear from the comments what part of the equation is supposed to do what
+*' * Variables and parameters should be declared along with a descriptive text (use `" "` for descriptive text to avoid compilation errors)
+*' * Use three asterisks `***` for comments or `*'` if the comment should show up in the documentation of REMIND 
+*' * Never use 4 asterisks (reserved for GAMS error messages)
+*' * Don't use the string `infes` in comments
+*' * Don't use `$+number` combinations, e.g., `$20` (this interferes with GAMS error codes).
+*' * Indicate the end of a file by inserting `*** EOF filename.inc ***` 
+*' 
+*' #### Sets
+*' 
+*' * don't use set element names with three capital letters (like `ETS` or `ESR`), otherwise the maglcass R library might interpret this as a region name when reading in GDX data
+*' 
+*' 
+*' #### Equations:
+*' The general idea is not to write code and equations as short as possible, but to write them in a way they can be read and understood as fast as possible. To that end:
+*' 
+*' * Write the mathematical operator (`*`, `/`, `+`, `-`) at the beginning of a line, not the end of the last line
+*' * Leave a space before and after `+` and `-` operators and equation signs (`=g=`, `=l=`, `=e=`)
+*' * Leave a space behind the comma of a sum (not behind the commas in set element calling)
+*' * Use indentations to make the structure more readable
+*' * Use full quotes (`"feel"`) instead of single quotes (`'feel'`) when specifying individual elements of a set (this makes automatic replacement via sed easier)
+*' * Put the equation sign (`=g=`, `=l=`, `=e=`) in a single line without anything else
```

Comment:
...except it is a very very short equation?

Reply:
"… unless the entire equation fits on one line"

Also, we should limit the line length.

Reply:
Fine with me, how do we best limit the line length in the code?

Reply:
80 characters is the universal limit accepted across [the](https://style.tidyverse.org/syntax.html#long-lines) [civilised](https://peps.python.org/pep-0008/#maximum-line-length) [world](https://www.kernel.org/doc/html/v4.10/process/coding-style.html#breaking-long-lines-and-strings).

Reply:
I meant _how_ in the sense of: do we enforce the limit via a pre-commit hook, via codeCheck, etc.? Agree that the limit should be 80 chars.

Reply:
We are not enforcing any of the rest of the coding etiquette either, apart from occasionally shaming people for breaking it.  

git hooks won't work, as they are local to the repositories.  `codeCheck()` could work, if somebody wants to implement it.  But there will be a hell of a backlog to deal with:
```
$ find ./ -name "*.gms" | xargs wc -L | sort -nr | head
    912 total
    912 ./modules/24_trade/standard/bounds.gms
    912 ./modules/24_trade/se_trade/bounds.gms
    912 ./modules/24_trade/capacity/bounds.gms
    805 ./modules/51_internalizeDamages/module.gms
    662 ./modules/50_damages/BurkeLike/realization.gms
    558 ./modules/51_internalizeDamages/KW_SEitr/realization.gms
    544 ./modules/29_CES_parameters/calibrate/realization.gms
    537 ./modules/16_downscaleTemperature/module.gms
    516 ./core/presolve.gms
```

Reply:
Ok, I think this conversation goes beyond the purpose of this PR, which was to avoid having the Coding Etiquette in two different places. So I'll just merge and we continue the discussion how to proceed with code line length somewhere else

Reply:
Well, enforcement aside, it should state somewhere in the etiquette lines should not go beyond 80 characters.